XamarinCommunityToolkit.UnitTests/Helpers/BindablePropertyBuilder_Tests.cs

```diff
@@ -13,7 +13,8 @@ public void InitializeWithPropertyNameShouldSetBindablePropertyPropertyName()
 		{
 			var propertyName = "adsf";
 
-			var bindableProperty = BindablePropertyBuilder.InitializeWithPropertyName(propertyName)
+			var bindableProperty = BindablePropertyBuilder.CreateBuilder()
```

Comment:
This is much better and very clear to the developer what they are doing. Thanks for making this change!